### DISCUSSION TITLE
Diffrent audio offset and auto audio offset for different output devices

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#27245] [Ideas]
Different output devices have a different latency would make sense i think. (i switch between speakers and headphones)

**Comment by Rekunan** [comment]
I know on a lot of mobile rhythm games they have separate presets for bluetooth and wired.

But I also just remembered that the devs plan on creating global setting presets/profiles, so that'll prob work for this.
